### PR TITLE
creating landing page

### PR DESCRIPTION
**Pull request by AnaCSilveira**
Creating HTML and CSS, including buttons, colors, title.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    <title>Calculadora</title>
+
+</head>
+<body>
+
+   
+    <main>
+
+        
+        <div id="screen">
```
Comment (assuncaocharles) [inline index.html]: Let's use class over id. It's really just a _nit_

File: style.css [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+width: 350px;
+margin: 80px auto;
+background-color: rgb(15, 7, 25);
+opacity: 0.8;
+height: 450px;
+border-radius: 8px;
+
+
+}
+input, .buttom{
```
Comment (assuncaocharles) [inline style.css]: Let's avoid using tags directly. replace the `input` selector with a meaningful class.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    font-size: 25px;
+}
+.buttons{
+    
+    display: flex;
+    justify-content: space-evenly;
+    
+        
+}
+button{
```
Comment (assuncaocharles) [inline style.css]: Same here about avoiding using tags.

File: style.css (hunk 2 above)
Comment (assuncaocharles) [reply style.css]: also you have a typo 
```suggestion
input, .button{
```

**Review by assuncaocharles — APPROVED** [review]
Once changes are done,
Good to merge

**Comment by AnaCSilveira** [comment]
Done.

**Pull request merged**